### PR TITLE
Update "usage" section of INSTALL file

### PR DESCRIPTION
**Pull request by otyshchenko1**
Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>

### CODE REVIEW

**Comment by otyshchenko1** [comment]
guys, any thoughts?

File: INSTALL [hunk 1]
```diff
@@ -214,4 +228,82 @@ all images on a storage device such as eMMC or SD card.
...
+cd workspace_base/meta-xt-prod-gen3-test/recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/
+
+- Edit domd-MACHINE_NAME.cfg to uncomment "extra" option which corresponds to NFS.
+- Edit domu-MACHINE_NAME.cfg to uncomment "extra" option which corresponds to NFS and comment "disk" option.
+
+So, as you can see, varying U-Boot's "boot_dev" and "bootcmd" environment variables
+and domain config's "extra" and "disk" options it is possible to choose  different boot device
+for each system component. For example, boot system using TFTP and then use SD card for guest domains.
+
+Remark regarding uirfs.sh script.
```
Comment (aanisov) [inline INSTALL]: I guess remarking about something requires this to be mentioned before.

Comment (otyshchenko1) [reply INSTALL]: ok. I will explain

**Comment by otyshchenko1** [comment]
@iartemenko probably merge this patch.

**Pull request merged**